src/page-building/career/career-page.tsx

```diff
@@ -0,0 +1,96 @@
+import { LayoutV2 } from '../../components/layout/layout-v2';
+import { Aurora } from '../../components/aurora/aurora';
+import { ContentBlockContainer } from '../../components/layout/content-block-container';
+import { PersonioJobPosition } from '../../@types/personio';
+import { Culture } from './culture';
+import { Perks } from './perks';
+import { LeadboxProps } from '../../new-components/leadbox/leadbox';
+import { useTranslation } from 'gatsby-plugin-react-i18next';
+
+const Paragraph = styled.p`
+  ${TextStyles.textR}
+  & + & {
+    margin-bottom: 16px;
+  }
+`;
+
+interface CareerPageProps {
+  positions: PersonioJobPosition[];
+}
+
+export const CareerPage = ({ positions }: CareerPageProps) => {
+  const { t } = useTranslation();
+
+  const leadbox: LeadboxProps = {
+    title: t('career.leadbox.title'),
+    illustration: 'astronaut_012',
+    contact: {
+      headline: t('career.leadbox.subtitle'),
+      title: t('career.leadbox.text'),
+      email: t('career.leadbox.mail'),
+    },
+  };
+
+  return (
+    <LayoutV2
+      leadbox={leadbox}
+      transparentHeader={true}
+      light={true}
+      hero={<Aurora />}
+    >
+      <ContentBlockContainer>
+        <SectionHeader
+          kicker="Karriere bei Satellytes"
+          headline="Arbeite mit uns"
+        >
+          <Paragraph>
+            Satellytes – das sind aktuell 14 ausschließlich leidenschaftliche
+            Entwickler:innen und Designer:innen. Wir haben großen Spaß an
+            Technologie und freuen uns auf neue Herausforderungen. Dabei
+            fokussieren wir uns auf langfristige Engagements im Konzerngeschäft.
+          </Paragraph>
+          <Paragraph>
+            Neuen Aufgaben begegnen wir immer mit angemessenem Respekt. Wir
+            streben stets hochwertige und zeitgemäße Lösungen an – die Wahl der
+            Technologie ist für uns dabei sekundär.
+          </Paragraph>
+          <Paragraph>
+            Unser Büro befindet sich in einem wunderschönen Altbau im Herzen
+            Münchens, in der Sendlinger Straße, unweit der gleichnamigen
+            U-Bahn-Station. In Laufweite gibt es nicht nur dutzende Läden,
+            Cafés, Restaurants und den bekannten Viktualienmarkt, sondern auch
+            diverse kleinere und größere Parks. Sogar die Isar ist in der
+            Mittagspause fußläufig oder mit der U-Bahn in nur 3 Minuten zu
+            erreichen.
+          </Paragraph>
+        </SectionHeader>
+      </ContentBlockContainer>
+
+      <ContentBlockContainer>
+        <ApplicationProcess />
+      </ContentBlockContainer>
+
+      <ContentBlockContainer>
+        <Openings jobs={positions} />
+      </ContentBlockContainer>
+
+      <ContentBlockContainer>
+        <Culture />
+      </ContentBlockContainer>
+
+      <ContentBlockContainer>
+        <Culture />
+      </ContentBlockContainer>
+
+      <ContentBlockContainer>
+        <Perks />
+      </ContentBlockContainer>
```

Comment:
I like the structure, each section of the career page is clearly visible. It's not a random mix of components anymore and if I want to update a place I can navigate straight to the right place 👍 

I don't like the noise coming from `ContentBlockContainer` and the extra hierarchy. This is just bringing in some spacing to the bottom. We can do better, why not so? (well I don't like that much either,  but it's a little bit better.

```
<ApplicationProcess />
<SectionSpace/>

<Openings jobs={positions} />
<SectionSpace/>

<Culture />
<SectionSpace/>

<Perks />
<SectionSpace/>
```

I don't want to merge the spacing into the sections itself. It mixes too much of the spacing concerns into an otherwise innocent component that you can check out easily in storybook right now (without any noisy spacings).

What we could do is to have some `SectionContainer` that is used by each of them (Perk, Culture etc) and then we have a rule like this:

```
& + & {
 margin-bottom..
```

That way they only add spacing between them if two or more are in the DOM. Maybe that's something?

Reply:
I'd still like to see a `<Stack />` approach in real life. `Stack` would be a component used purely for layouting other components.

I have something like this in mind:
Usage:
```html
<Stack [space="small|medium|large"]>
  <Openings jobs={positions} />
  <Culture />
  <Perks />
</Stack>
```
Styles inside `<Stack>`:
```css
:host > *:not(:last-child) {
  margin-bottom: var(--...);
}
```

Reply:
yeah that's nice too 🤘 But for this we need strict rules and the knowledge that we have instead distinct sections everywhere like I created here.

Reply:
I'm fine with sticking to the `ContentBlockContainer` that you introduced for now. Nevertheless, we certainly need a more accessible solution for the future.

Reply:
Just read this article. Seems like I missed the point where spacer components were cool again 😄 

https://www.joshwcomeau.com/react/modern-spacer-gif/

The article has some good arguments for `<Spacer />`. I guess we need to see how exactly we will use the component. For example, if we iterate over each content block, putting each block into a `<ContentBlockContainer />` feels better then returning `<><Culture /><Spacer/></>`. 

A `Stack` component is something I've never seen in the wild. Also sounds interesting as it is less verbose than both other solutions.  Without digging too deep into it, I wonder if it really works very reliable, as it relies on a certain structure of the children.